tests/unit/amo/components/TestAddonReviewListItem.js

```diff
@@ -197,6 +198,14 @@ describe(__filename, () => {
     expect(root.find('.AddonReviewListItem-edit')).toHaveLength(0);
   });
 
+  it('assigns isOwner property if you wrote the review', () => {
+    const review = signInAndDispatchSavedReview();
```

Comment:
I think it'd be better to be explicit about the IDs used here, as is done elsewhere in the files. This works but is not as obvious to read. You can change it to:

```js
const review = signInAndDispatchSavedReview({
  siteUserId: 123, reviewUserId: 123,
});
```

and it's a lot easier to read.